### ISSUE TITLE
Components API is replacing terms in BEL statements

### ISSUE THREAD

**Issue by maestroe** [OpenBEL/openbel-api#142]
Calling /components API for the BEL statement: 
```
a(CHEBI:heparin) increases complex(p(MGI:Serpind1),p(MGI:F2))
```
returns result for
```
a(CHEBI:"heparan sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine") -> complex(p(MGI:Serpind1),p(MGI:Rpl3))
```

The API replaced two of the following valid terms:
 - CHEBI:heparin
 - MGI:F2

This makes the API extremely unstable.

Reproduce:
wget -q -O - "https://{HOST}/api/expressions/a%28CHEBI%3Aheparin%29%20increases%20complex%28p%28MGI%3ASerpind1%29%2Cp%28MGI%3AF2%29%29/components" | jq
```
{
  "expression_components": {
    "subject": {
      "term": {
        "function": {
          "short": "a",
          "long": "abundance",
          "return_type": "abundance",
          "signatures": [
            "abundance(E:abundance)abundance"
          ],
          "description": "Denotes the abundance of an entity"
        },
        "arguments": [
          {
            "parameter": {
              "namespace": "CHEBI",
              "value": "heparan sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine"
            }
          }
        ]
      }
    },
    "relationship": {
      "short": "->",
      "long": "increases",
      "description": "Increases:  +A -> B+ -  For terms A and B, +A increases B or\nA -> B+ indicates that increases in A have been observed to\ncause increases in B. +A increases B+ also represents cases\nwhere decreases in A have been observed to cause decreases in B,\nfor example, in recording the results of gene deletion or other\ninhibition experiments. A is a BEL Term and B is either a BEL\nTerm or a BEL Statement. The relationship does not indicate\nthat the changes in A are either necessary for changes in B,\nnor does it indicate that changes in A are sufficient to cause\nchanges in B.\n"                                                                                                            
    },
    "object": {
      "term": {
        "function": {
          "short": "complex",
          "long": "complexAbundance",
          "return_type": "complexAbundance",
          "signatures": [
            "complexAbundance(F:abundance...)complexAbundance",
            "complexAbundance(E:complexAbundance)complexAbundance"
          ],
          "description": "Denotes the abundance of a molecular\n          omplex"
        },
        "arguments": [
          {
            "term": {
              "function": {
                "short": "p",
                "long": "proteinAbundance",
                "return_type": "proteinAbundance",
                "signatures": [
                  "proteinAbundance(E:proteinAbundance,F:proteinModification)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:substitution)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:truncation)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:fusion)proteinAbundance"
                ],
                "description": "Denotes the abundance of a protein"
              },
              "arguments": [
                {
                  "parameter": {
                    "namespace": "MGI",
                    "value": "Serpind1"
                  }
                }
              ]
            }
          },
          {
            "term": {
              "function": {
                "short": "p",
                "long": "proteinAbundance",
                "return_type": "proteinAbundance",
                "signatures": [
                  "proteinAbundance(E:proteinAbundance,F:proteinModification)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:substitution)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:truncation)proteinAbundance",
                  "proteinAbundance(E:proteinAbundance,F:fusion)proteinAbundance"
                ],
                "description": "Denotes the abundance of a protein"
              },
              "arguments": [
                {
                  "parameter": {
                    "namespace": "MGI",
                    "value": "Rpl3"
                  }
                }
              ]
            }
          }
        ]
      }
    }
  },
  "statement_short_form": "a(CHEBI:\"heparan sulfate D-glucuronyl-D-galactosyl-D-galactosyl-D-xylosyl-L-serine\") -> complex(p(MGI:Serpind1),p(MGI:Rpl3))"
}
```

**Comment by wshayes** [comment]
Can get Tony Bargnesi to confirm, but the resulting terms are the preferred/official terms for the namespaces for those entities. 

Rpl3 is the official MGI symbol for F2

I’m not sure why the IUPAC name is replacing the heparin label in CHEBI - Tony would know.  Most likely that is an issue with the terminology database.

**Comment by abargnesi** [comment]
resolved and on `master`, @wshayes can you make patch release?

**Comment by wshayes** [comment]
Thank Tony, I'll do that this weekend.

**Comment by maestroe** [comment]
@wshayes and @abargnesi thanks for changes and the deployment.

@wshayes are you sure about "Rpl3 is the official MGI symbol for F2"?

Rpl3 do has a synonym called "F2". But the symbols (concepts) F2 and Rpl3 still both exists in MGI (also in HGNC) and have a different chromosome, location information as well as different Ids.

Rpl3: http://www.informatics.jax.org/marker/MGI:1351605
F2: http://www.informatics.jax.org/marker/MGI:88380

**Comment by wshayes** [comment]
@maestroe My bad - should have checked further.  I'm still working on getting the patch out and deployed.

**Comment by wshayes** [comment]
Patch deployed
